Forge/src/main/java/me/ferdz/placeableitems/network/SUpdateFluidHolderPacket.java

```diff
@@ -0,0 +1,81 @@
+package me.ferdz.placeableitems.network;
+
+import me.ferdz.placeableitems.tileentity.FluidHolderTileEntity;
+
+import net.minecraft.network.PacketBuffer;
+import net.minecraft.util.ResourceLocation;
+import net.minecraft.util.math.BlockPos;
+
+import net.minecraftforge.fluids.FluidStack;
+import net.minecraftforge.registries.ForgeRegistries;
+
+/**
+ * A server -> client packet to update the state of a {@link FluidHolderTileEntity}.
+ *
+ * @author Parker Hawke - Choco
+ */
+public class SUpdateFluidHolderPacket {
```

Comment:
What does the S prefix stand for?

Reply:
It's a client-bound packet (sent from the server - hence `S`). Forge's mappings follow this same nomenclature.